mantra/core/bot.py

```diff
@@ -29,6 +33,7 @@ def run_bot(self) -> None:
         super().run(asyncio_debug=True)
 
     async def on_starting(self, event: hikari.StartingEvent) -> None:
+        asyncio.create_task(self.establish_db_connection())
```

Comment:
Just like establishing a connection, it is equally important to close the connections when exiting the bot.
Please add the functionality for that